app/google-services.json

```diff
@@ -1,3 +1,351 @@
 {
-
+  "project_info": {
```

Comment:
delete this file.
In Readme.md, add this statement to configure the project with suitable google-service.json file by creating a project on Firebase

Reply:
Done